bundle_conda.py

```diff
@@ -221,14 +241,16 @@ def _constructor(version=_version(), extra_specs=None):
         "initialize_by_default": False,
         "license_file": os.path.join(HERE, "resources", "bundle_license.rtf"),
         "specs": base_specs,
-        "extra_envs": {f"napari-{version}": {"specs": napari_specs}},
+        "extra_envs": {f"napari-{version}": napari_env_config},
         "menu_packages": [
             "napari-menu",
         ],
         "extra_files": {
             "resources/bundle_readme.md": "README.txt",
             empty_file.name: ".napari_is_bundled_constructor",
             condarc: ".condarc",
+            # Maybe this should be a package...?
```

Comment:
I just thought it would be a nice-to-have option in conda-forge. No need to hack around with files; just provide the package and install normally. Helps with provenance too. `ocl-icd-system` is a similar (activation-only) package.